auth/bitbucket/cloud/auth_config.go

```diff
@@ -0,0 +1,64 @@
+package cloud
+
+import (
+	"errors"
+	"fmt"
+	"github.com/concourse/atc"
+	"github.com/concourse/atc/auth/bitbucket"
+	"github.com/concourse/atc/auth/routes"
+	"github.com/hashicorp/go-multierror"
+	"github.com/tedsuo/rata"
+)
+
+type AuthConfig struct {
+	ClientID     string `json:"client_id" long:"client-id" description:"Application client ID for enabling Bitbucket OAuth"`
+	ClientSecret string `json:"client_secret" long:"client-secret" description:"Application client secret for enabling Bitbucket OAuth"`
+
+	Users        []string                     `json:"users,omitempty" long:"user" description:"Bitbucket users that are allowed to log in"`
+	Repositories []bitbucket.RepositoryConfig `json:"repositories,omitempty" long:"repository" description:"Bitbucket repositories whose members are allowed to log in"`
```

Comment:
Just to confirm, this currently does nothing?

Reply:
Yup. We we're still working out the differences between Bitbucket Cloud and Server. This should be functional before an eventual merge, but we wanted feedback on the rest of the pull request first.